### PR TITLE
Switch to astropy build and add pip to conda dependency

### LINKED ISSUE #85: Adding pip to environment.yml
**Describe the bug**

Environment.yml file needs an extra pip dependency.

**Expected behavior**

Calling `conda create -f environment.yml python=3.6` should run fine.

**Actual behavior**

This gets added at the top of the output: "Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you."

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Ubuntu 20.04 LTS w/ WSL2
- Which version of Python (`python -V`)? 3.7, latest conda install.

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #85, #82 

 - Switches over the travis.yml file from doing a simple miniconda install to using astropy's ci-helper.
 - Also a very very minor change, but might as well make conda happy :)

### CODE REVIEW

**Comment by IAlibay** [comment]
Note; don't merge for now, I've just pushed some change to one of the transformations ipython blocks to see if this was the source of the segfault. Either way I'll probably revert that last commit.

**Comment by IAlibay** [comment]
Ok well somehow it works, I don't know what astropy's helpers do, but it's like magic....

**Comment by IAlibay** [comment]
@lilyminium I've cleaned up Travis config, and added the extra pip to the environment file. It's a little bit messier than before, but at least now it  _should_ compile properly so it might be a good temporary measure to keep the userguide going.

File: .travis.yml [hunk 1]
```diff
@@ -1,27 +1,39 @@
...
   global:
   - secure: QM4CHEqPdDEUhrQYvkOz4DqxNjUNHLVRkxCSy4raFgTEAUyiPF+nF2z8Q1s8lfTxBwmjgEgLpxBMCRI2wbkLJirHwAdpvkH3aNzla4o2mgNdSyl1zSQrflM8/eJ3uEQjx69dcfPQWmED1hQQceh+Gd2c9hhfDdPv12viqHhFp5/TEeXRILHczbE0bC8efxSI87xi5a6rCm6c+Uohmc7Xy5+PraVsuxrRWU7NESqTYuIr14gT8p38SvwCafALpYw7i0Ch1ue/ffrsPKlnnCGd/HTE7YMcw61dUh9Y8QYTSJaBBdLS8ytwTyaN4/4LEEqaHihwKko8j03hUi+TM+d/lmKYh3y3U7U3BaofW/aGJu5Xuo6eKtgwA2WX/yTLdVQbnMgDOtJiKQ7G4wAKO4UUffRbdUc3ioOKBPEPjhyQ/Qo3NEozGt0SUNzEdeFIMaQbBQzxzGy+NNByPnOlvPK4u6hvAluPPVqGaVV7gAh5NRzPrAoChK2TYASZVCd9SQ8HfnFdv5aWn/rt6wnf2wVRcCxtQXSKdazaLFv1noSO44PWprRNskI2xYEjrFRMk/Drk7uNBCcJH0apBbDyAzaQ+izjyWllGztrebjQcJSagfhES/Rt6WBjuwAVzElHwRgIWxrGn1t0MzT3q7IfAQlXedRhGyv4e+dxehjZdv7m4W8=
   - SPHINX_DIR=doc/
   - HTML_DIR=doc/build/html
+  - PYTHON_VERSION=3.6
+  - CONDA_DEPENDENCIES="jupyter_contrib_nbextensions nglview sphinx ipywidgets MDAnalysis MDAnalysisTests plotly"
+  - CONDA_CHANNELS='defaults conda-forge plotly'
+  - PIP_DEPENDENCIES="sphinx_rtd_theme sphinx-sitemap nbsphinx ipython tabulate sphinxcontrib-bibtex pybtex"
+  - CYTHON_TRACE_NOGIL=1
+  - MPLBACKEND=agg
```
Comment (IAlibay) [inline .travis.yml:19]: This one will be necessary from MDA 1.0.1 so I added it pre-preemptively.

File: .travis.yml [hunk 2]
```diff
@@ -1,27 +1,39 @@
...
 - '3.6'
+
+
 env:
   global:
   - secure: QM4CHEqPdDEUhrQYvkOz4DqxNjUNHLVRkxCSy4raFgTEAUyiPF+nF2z8Q1s8lfTxBwmjgEgLpxBMCRI2wbkLJirHwAdpvkH3aNzla4o2mgNdSyl1zSQrflM8/eJ3uEQjx69dcfPQWmED1hQQceh+Gd2c9hhfDdPv12viqHhFp5/TEeXRILHczbE0bC8efxSI87xi5a6rCm6c+Uohmc7Xy5+PraVsuxrRWU7NESqTYuIr14gT8p38SvwCafALpYw7i0Ch1ue/ffrsPKlnnCGd/HTE7YMcw61dUh9Y8QYTSJaBBdLS8ytwTyaN4/4LEEqaHihwKko8j03hUi+TM+d/lmKYh3y3U7U3BaofW/aGJu5Xuo6eKtgwA2WX/yTLdVQbnMgDOtJiKQ7G4wAKO4UUffRbdUc3ioOKBPEPjhyQ/Qo3NEozGt0SUNzEdeFIMaQbBQzxzGy+NNByPnOlvPK4u6hvAluPPVqGaVV7gAh5NRzPrAoChK2TYASZVCd9SQ8HfnFdv5aWn/rt6wnf2wVRcCxtQXSKdazaLFv1noSO44PWprRNskI2xYEjrFRMk/Drk7uNBCcJH0apBbDyAzaQ+izjyWllGztrebjQcJSagfhES/Rt6WBjuwAVzElHwRgIWxrGn1t0MzT3q7IfAQlXedRhGyv4e+dxehjZdv7m4W8=
   - SPHINX_DIR=doc/
   - HTML_DIR=doc/build/html
+  - PYTHON_VERSION=3.6
+  - CONDA_DEPENDENCIES="jupyter_contrib_nbextensions nglview sphinx ipywidgets MDAnalysis MDAnalysisTests plotly"
```
Comment (IAlibay) [inline .travis.yml:15]: I've unpined the MDA dependencies from 0.20.1, can re-pin if necessary.

File: .travis.yml [hunk 3]
```diff
@@ -1,27 +1,39 @@
 language: python
+group: travis_latest
 cache: pip
 sudo: false
 python:
 - '3.6'
```
Comment (IAlibay) [inline .travis.yml:6]: I'm not sure this is required, kept it in case you thought otherwise.

File: .travis.yml (hunk 2 above)
Comment (lilyminium) [reply .travis.yml:15]: I'm curious, would this work with [CONDA_ENVIRONMENT](https://github.com/astropy/ci-helpers#travis) as well? That would help with redundancy.

Comment (IAlibay) [reply .travis.yml]: Unfortunately it seems to cause travis to crash (bcf55c0).

**Pull request merged**